dotnet/samples/Demos/ContentSafety/Exceptions/TextModerationException.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace ContentSafety.Exceptions;
+
+public class TextModerationException : Exception
```

Comment:
Can you add some API documentation.

Is there more information about the attack that can be included in this exception?

Reply:
Added a documentation and extended this exception to include analysis details per category:
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/bdeeb1ad-336f-4718-8376-a464624d6cee)